src/input/NewButton.tsx

```diff
@@ -0,0 +1,77 @@
+import { Button as MaterialUiButton, Grid, makeStyles, Avatar, Typography } from '@material-ui/core';
+import React from 'react';
+
+
+interface TileButtonProps {
+    title?: string
+    subTitle?: string
+    percent?: string
+    backgroundColor?: string
+    color?: string
+    selected?: boolean
+}
+
+const useStyles = makeStyles((theme) => ({
+    buttonSelected: {
+        backgroundColor: 'white',
+        color: '#38b2cc',
+        width: '9em',
+        height: '5em'
+    },
+    buttonNotSelected: {
+       backgroundColor: 'rgba(56,178,204,0.4)',
+        color: '#38b2cc',
```

Comment:
I suggest to add 

opacity: '60%'

it makes it look more like it is not selected.